postcss.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = () => ( {
+	plugins: {
+		'postcss-custom-properties': {
+			importFrom: [ require.resolve( '@automattic/calypso-color-schemes' ) ],
```

Comment:
I think it would make sense to use [`@automattic/color-studio`](https://github.com/Automattic/color-studio/) directly here.

I don't really see Calypso colors bringing any additional value; it just has some extra Calypso specific things such as `--sidebar-background`, `--masterbar-toggle-drafts-editor-background`, etc...

Unless I'm missing something?

Reply:
Was considering that, yeah. I think we'd have to rename a few colors though, since they're aliased in `@automattic/calypso-color-schemes`.

Furthermore, I think there's still some issue that prevents things from working entirely as they should, see https://github.com/Automattic/jetpack/pull/13854#issuecomment-550898168 :thinking:

Reply:
>  I think we'd have to rename a few colors though, since they're aliased in `@automattic/calypso-color-schemes`.

That might not have been very clear; what I meant is that we're right now using the names as found in `@automattic/calypso-color-schemes` in Jetpack (not names from `@automattic/color-studio`).

Reply:
> I don't really see Calypso colors bringing any additional value

`calypso-color-schemes` brings exactly the value that the package name promises: it defines a family of color schemes like Classic Blue, Midnight or Powder Snow that can be applied to UI that uses "semantic" names like `color-primary`, `color-accent` or `masterbar-item`.

`color-studio` is a generic set of named colors (somewhat similar to RAL or Pantone).

If you want to style your UI with a color scheme that was designed to work together, then you need one of the Calypso color schemes.

Reply:
Thanks for clarification!

Sounds like @Automattic/jetpack-design could have final say which one they prefer to work with?

Reply:
I'm not entirely sure of the question. It seems a library choice question... what's the difference from a design perspective?

If it answer the question: going forward we'll use the latest `color-studio` colors, which includes the new Jetpack green and a gamut of colors agreed across all A8c.

Reply:
@folletto I think the question is more like: does Jetpack have something like "color scheme" that is applied to the UI? And does it intend to share the default color scheme with Calypso, or does it define its own?

Color scheme defines approx 15-20 colors, all selected from the `color-studio` palette, while the whole `color-studio` has 170+ colors.

Reply:
Not explicitly. I think the code we have already there is the closest thing we have to a color scheme.

For sure, we won't use all the colors in the `color-studio`, but right now there isn't a logical subset that has been extracted — again, except what's already there in code.